plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java

```diff
@@ -1350,9 +1474,9 @@ protected Composite createConnectionDbTypeEMFComboViewer(FormToolkit widgetFacto
 			 * 	
 			 */
 			public void selectionChanged(SelectionChangedEvent event) {
-				if (propertiesEditionComponent != null)
-					propertiesEditionComponent.firePropertiesChanged(new PropertiesEditionEvent(DBLookupMediatorPropertiesEditionPartForm.this, EsbViewsRepository.DBLookupMediator.Connection.connectionDbType, PropertiesEditionEvent.COMMIT, PropertiesEditionEvent.SET, null, getConnectionDbType()));
-				fillDbConnectionDefaultValues();
+//				if (propertiesEditionComponent != null)
```

Comment:
Shall we uncomment this code fragment if it is necessary or delete if it is unnecessary?

Reply:
fixed